File: Service_eHospital_DongNai_A_Config/Implement/Sys_ModifySource_ClientService.cs
namespace Service_eHospital_DongNai_A_Config.Implement
{
    public class Sys_ModifySource_ClientService : BaseService<Sys_ModifySource_Client, ISys_ModifySource_ClientRepository>
    , ISys_ModifySource_ClientService
    {
    private readonly ISys_ModifySource_ClientRepository _Sys_ModifySource_ClientRepository;
    public Sys_ModifySource_ClientService(ISys_ModifySource_ClientRepository Sys_ModifySource_ClientRepository) : base(Sys_ModifySource_ClientRepository)
    {
    _Sys_ModifySource_ClientRepository = Sys_ModifySource_ClientRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A_Config/Sys_ModifySource_Client.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_ModifySource_Client } from './Sys_ModifySource_Client.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_ModifySource_ClientService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'DateUpdate', 'VersionNo', 'ComputerName', 'FileName', 'PathName', 'Size', 'ModifyDate'];

    List: Sys_ModifySource_Client[] | undefined;
    ListFilter: Sys_ModifySource_Client[] | undefined;
    FormData!: Sys_ModifySource_Client;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_ModifySource_Client";
    }
}

